src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/operations/smugglerDatabaseDetails.ts

```diff
@@ -300,19 +302,21 @@ class smugglerDatabaseDetails extends abstractOperationDetails {
             }
         }
 
+        let processingSpeedText = smugglerDatabaseDetails.ProcessingText;
```

Comment:
what's the purpose of this variable?

Reply:
As discussed - static text used in a few places